### PR TITLE
feat: add playground

### PR DESCRIPTION
**Pull request by Weetbix**
This PR adds a local development playground which shows the various metric scores.

To run, make sure you `yarn` and then run `yarn playground:dev`

Changes:
- Add playground
- Split readability.ts filesystem usage to another file so that we can import the original file in a web project (no node usage with `glob`).

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ jobs:
 | `build`     | build the dist file. You are required to run this locally in order to build the dist before opening a PR.                                                                                                                                       |
 | `run:local:report` | Run the action on two local folders, for testing and development. The first argument is the old ("base branch") folder and the sedcond argument is the new ("head branch") folder. For example `yarn run:local:report ./test-data/old ./test-data/new` |
 | `run:local:debugfile` | Show what the program parses from a single file, after stripping all non-wanted items from the file, and before rating the readability. For example `yarn run:local:debugfile ./test-data/new/test-document.md` |
+| `playground:dev` | Open the playground in local development mode for testing readability scores |
```
Comment (noon-dawg) [inline README.md:97]: I think the decrease in readability is because this sentence doesn't end with a period

**Review by noon-dawg — DISMISSED** [review]
This is awesome!!! 🚀 🚀 🚀

File: README.md (hunk 1 above)
Comment (Weetbix) [reply README.md]: I added periods to them all and the score didn't change.

**Review by noon-dawg — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (noon-dawg) [reply README.md]: I'm not sure what decreased the score then 😕

Comment (Weetbix) [reply README.md]: Its a small amount of content so I think the sore can be sway easily by the addition or removal of a few lines.

**Pull request merged**